app/views/hyrax/collections/_showcase_tile.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="thumbnail">
+    <div id="image-<%= index %>" onmouseover='toggleTitleOn("<%= index %>")' onmouseout='toggleTitleOff("<%= index %>")'>
```

Comment:
Here's some thoughts on hover functionality:
We CAN do it with JS and that's probably easier conceptually and maybe for maintenance. 
But we could also use the `:hover` CSS selector to change CSS, rather than add/remove style/classes with mouse events. This will work more consistently across browsers and may perform better.

Something like this should work:
```
img.hoverable {
  border: none;
  &:hover { // & Means the parent selector. Simplifies to "img.hoverable:hover"
    border: 5px solid $black;
  }

  & + .show-me { // + Means sibling to. Simplifies to "img.hoverable (find adjacent) .show-me"
    margin-top: -1em; // Just assuming the margins can be tweaked down to em relative, not pixel perfect. Not a big deal if not, may cause mobile problems if using pixels?
    margin-left: -2em;
    color: white;
    display: none;
  }
  &:hover + .show-me {
    display: block;
  }
}
```

I don't know at this point if where you'd want to put that CSS and if we want to make it more specific or if it's going to be used again for similar cards. I think it could be used again on the search results pages.

Reply:
Resolved. this is ok but we can look back and see what pieces of the app could benefit from being JS-less down the road to help further with accessibility.